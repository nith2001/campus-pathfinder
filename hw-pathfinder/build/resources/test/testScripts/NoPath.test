# Find a path with Dijkstra's algorithm when
# the path found by BFS is different.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A B 1.0
AddEdge simpleGraph C D 1.0

ListNodes simpleGraph

FindPath simpleGraph A C

FindPath simpleGraph B D

FindPath simpleGraph A B

FindPath simpleGraph C D
