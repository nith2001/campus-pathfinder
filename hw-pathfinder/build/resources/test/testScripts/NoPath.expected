# Find a path with Dijkstra's algorithm when
# the path found by BFS is different.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 1.000 from A to B in simpleGraph
added edge 1.000 from C to D in simpleGraph

simpleGraph contains: A B C D

path from A to C:
no path found

path from B to D:
no path found

path from A to B:
A to B with weight 1.000
total cost: 1.000

path from C to D:
C to D with weight 1.000
total cost: 1.000